style.css

```diff
@@ -532,22 +556,45 @@ textarea:hover::placeholder {
   }
 
   #about-me {
-    background-image: url(images/illustration\ about\ me\ desktop.jpg);
-    background-repeat: no-repeat;
-    background-size: cover, auto;
-    background-position: center center;
+    background-image: none;
+    gap: 2rem;
+  }
+
+  #background-about {
+    width: 115%;
+    margin: 5rem;
+    padding-left: 0%;
+    padding-right: 0%;
+    margin-left: 0%;
+    margin-right: 0%;
   }
 
   #about-me-button {
     display: flex;
     flex-direction: row;
-    flex-direction: column;
     align-items: center;
     justify-content: center;
-    padding: 4%;
+    padding: 14px;
     margin: 0 auto;
   }
 
+  .about-me-text {
+    width: 780px;
+    padding: 0%;
+    margin: 0%;
+  }
+
+  #about-img-1 {
+    max-width: 150px;
+    float: right;
+  }
+
+  #about-img-2 {
+    max-width: 150px;
+    float: left;
+    left: -10px;
+  }
```

Comment:
- [ ] You have done a great job to add images 👏 . however, you should notice that the image in the design is for decoration purposes. The images that are added to improve the UI of the application should be added as background images. Therefore I suggest that you add this image as a background image instead.
- [ ] In addition, I suggest that the image coves 100% width of the screen size. This is to ensure that you match the original design as much as possible ☺️.

Your design 😎 
![image](https://github.com/danielf-canon/Portfolio/assets/99394374/23207196-52b8-46d5-88ae-74ca1e1cc5ab)

Original design 😎 
![image](https://github.com/danielf-canon/Portfolio/assets/99394374/e772e0d5-cd25-4e70-96da-833ff2792167)